src/transfer_context.h

```diff
@@ -21,11 +23,20 @@ class TransferContext : public JavaInstanceWrapper {
     void write_args(jni::Env& p_env, const Vector<KtVariant>& p_args);
     Vector<KtVariant> read_args(jni::Env& p_env, bool p_refresh_buffer);
 
+    static void initVariantArgsPtrs();
+
+    static void icall(JNIEnv* rawEnv, jobject instance, jlong jPtr,
+               jstring jClassName, jstring jMethod,
+               jint expectedReturnType, bool p_refresh_buffer);
+
 private:
     SharedBuffer* get_buffer(jni::Env& p_env, bool p_refresh_buffer);
     bool ensure_capacity(jni::Env& p_env, long p_capacity);
     SharedBuffer* shared_buffer;
-};
 
+    static Variant variantArgs[MAX_ARGS_SIZE];
```

Comment:
do we need to declare them here? Can it be an implementation detail of the cpp file?